packages/repository-json-schema/src/__tests__/unit/build-schema.unit.ts

```diff
@@ -232,13 +232,18 @@ describe('build-schema', () => {
       expect(key).to.equal('modelPartial');
     });
 
+    it('returns "excluding[]" when a single option "exclude" is set', () => {
+      const key = buildModelCacheKey({exclude: []});
```

Comment:
Please use a non-empty list of excluded properties, to verify how are the properties encoded in the key.

Reply:
```suggestion
      const key = buildModelCacheKey({exclude: ['id', '_rev']});
```